scripts/rollup.config.js

```diff
@@ -0,0 +1,34 @@
+const generate = require('videojs-generate-rollup-config');
+const createTestData = require('./create-test-data.js');
+
+// see https://github.com/videojs/videojs-generate-rollup-config
+// for options
+
+const shared = {
+  primedPlugins(defaults) {
+    return Object.assign(defaults, {
+      createTestData: createTestData()
+    });
+  },
+  plugins(defaults) {
+    defaults.test.splice(0, 0, 'createTestData');
+
+    return defaults;
+  }
+};
+const mainBuilds = generate(Object.assign({input: 'lib/index.js', distName: 'mux'}, shared)).builds;
+const mp4Builds = generate({input: 'lib/mp4/index.js', distName: 'mux-mp4'}).builds;
+const flvBuilds = generate({input: 'lib/flv/index.js', distName: 'mux-flv'}).builds;
```

Comment:
What's being used for the UMD name here? Looks like previously it was set to `muxjs`.

Reply:
updated to muxjs